tools/postman/empty apd write command.json

```diff
@@ -0,0 +1,44 @@
+[
```

Comment:
My personal preference is to replace spaces with dashes `-` in filenames, so that I don't ever have to quote a path or use escape characters to navigate to a file at the command line.

Sorry if I'm "picking nits".

https://superuser.com/questions/29111/what-technical-reasons-exist-for-not-using-space-characters-in-file-names